_includes/markdown/JavaScript.md

```diff
@@ -283,23 +287,21 @@ Notice how, in cached versions, we are pulling the menu selection out of the eve
 Event delegation is the act of adding one event listener to a parent node to listen for events bubbling up from its children. This is much more performant than adding one event listener for each child element. Here is an example:
 
 ```javascript
-document.getElementById( 'menu' ).addEventListener( 'click', ( e ) => {
-
-    const currentTarget = e.currentTarget;
-    let target = event.target;
-
-    if ( currentTarget && target ) {
-      if ( 'LI' === target.nodeName ) {
-        // Do stuff with target!
-      } else {
-        while ( currentTarget.contains( target ) ) {
-          // Do stuff with a parent.
-          target = target.parentNode;
-        }
-      }
-    }
-
-} );
+document.getElementById('menu').addEventListener('click', (e) => {
+	const { currentTarget } = e;
+	let { target } = event;
```

Comment:
Shouldn't these both be `e` or `event`? Seems like `event` is currently undefined.

Reply:
Good catch! I'll update this.